### PR TITLE
Updating Dogwood release notes

### PR DESCRIPTION
**Pull request by nedbat**
- Use subsections rather than a table for point releases
- Add Dogwood.2 release notes, that need details added.

### CODE REVIEW

**Comment by nedbat** [comment]
@lamagnifica are you ok with the new format? Do you have details of the cert fix? And how do we normally note security fixes?

File: en_us/open_edx_release_notes/source/dogwood.rst [hunk 1]
```diff
@@ -496,19 +496,25 @@ release of Open edX.
...
-   * - Date
-     - Change
-   * - 9 March 2016
-     - An update to the release packaging was made.
-   * -
-     - The help links in Studio now open topics in the correct version of the
-       *Building and Running an Open edX Course* guide.
+* Django is updated to version 1.8.12.
 
+* Certificates were fixed in some way???
```
Comment (lamagnifica) [inline en_us/open_edx_release_notes/source/dogwood.rst]: from https://github.com/edx/edx-documentation/pull/956/files:
The `AUDIT_CERT_CUTOFF_DATE` feature flag is now available. For more
information, see :ref:`installation:Discontinue Audit Certificates` in
the _Building and Running an Open edX Course_ guide.

**Comment by lamagnifica** [comment]
@nedbat For security fixes, less is usually more. I suggest checking in with DevOps/Security team to see if they're ok with a public announcement.

**Comment by lamagnifica** [comment]
I will eliminate the change to the release notes that I made in my PR, and leave yours to make all of the updates for Dogwood.2

**Comment by nedbat** [comment]
@lamagnifica how about now?

**Comment by lamagnifica** [comment]
:+1: with super nit, "were fixed" --> "are fixed" would make the verbs parallel in each bullet...

**Comment by nedbat** [comment]
Nit fixed, thanks, Super-Nit!
index

**Comment by lamagnifica** [comment]
damn, can I change my username???

**Pull request merged**